target/linux/lantiq/ase/config-5.15

```diff
@@ -15,7 +14,7 @@ CONFIG_LANTIQ_ETOP=y
 CONFIG_NLS=y
 CONFIG_SGL_ALLOC=y
 CONFIG_SOC_AMAZON_SE=y
-CONFIG_SOC_TYPE_XWAY=y
+# CONFIG_SOC_XWAY is not set
```

Comment:
I would leave CONFIG_SOC_TYPE_XWAY enabled on the ase subtarget.

Reply:
The `CONFIG_SOC_TYPE_XWAY` was moved from the subtarget configs to the target config by @hauke:
https://github.com/openwrt/openwrt/pull/15233/commits/7ce95c34b9528ca0f7ddf38f1ef97131816548fb#diff-4674295b90225b63952a5d5a76b57933a6324e19ce2cc8031bee5cf8e8c54c1aR148

Reply:
You're right, but this is overridden by disabling this symbol on the subtarget.

Reply:
No, you have to look closely. These are 2 different symbols: 
`CONFIG_SOC_TYPE_XWAY` and
`CONFIG_SOC_XWAY`.

Reply:
@hauke also noted in his commit message that after this refresh the resulting .configs in the kernel tree are still unchanged.